spec/controllers/lists_controller_spec.rb

```diff
@@ -142,6 +142,12 @@
                     "/new-list",
                     "/newer-list",
                   ],
+                  "content_ids" => %w[
+                    123
+                    456
```

Comment:
Similar point about above where some of the items in the above list use variables rather than hardcoding (not sure why it's only some). Again similar comments for changes below.